### PR TITLE
ability to set group and item draggability independently

### PR DESCRIPTION
**Pull request by corey-fuller**

### CODE REVIEW

**Comment by appflowy** [comment]
@corey-fuller  Thanks for contribution. I will check it.

File: lib/src/widgets/board_data.dart [hunk 1]
```diff
@@ -205,6 +205,13 @@ class AppFlowyBoardController extends ChangeNotifier
   void updateGroupItem(String groupId, AppFlowyGroupItem item) {
     getGroupController(groupId)?.replaceOrInsertItem(item);
   }
+  
+  void setGroupDraggability(bool groupDragging, itemDragging) {
+    for (var groupController in _groupControllers.values) {
```
Comment (Xazin) [inline lib/src/widgets/board_data.dart:210]:
```suggestion
    for (final groupController in _groupControllers.values) {
```

File: lib/src/widgets/board_group/group_data.dart [hunk 2]
```diff
@@ -171,6 +171,15 @@ class AppFlowyGroupController extends ChangeNotifier with EquatableMixin {
         -1;
   }
 
+  void setDraggability(bool groupDragging, itemDragging) {
+    groupData.draggable = groupDragging;
+
+    for (var item in groupData._items) {
```
Comment (Xazin) [inline lib/src/widgets/board_group/group_data.dart:177]:
```suggestion
    for (final item in groupData._items) {
```

File: lib/src/widgets/board_group/group_data.dart [hunk 3]
```diff
@@ -171,6 +171,15 @@ class AppFlowyGroupController extends ChangeNotifier with EquatableMixin {
         -1;
   }
 
+  void setDraggability(bool groupDragging, itemDragging) {
```
Comment (Xazin) [inline lib/src/widgets/board_group/group_data.dart:174]: Can we add document comment for this method,.

**Review by Xazin — CHANGES_REQUESTED** [review]

**Comment by Xazin** [comment]
Hey @corey-fuller, sorry about the delay.

Are you interested in finishing this PR? I've added some minor comments above, and I'd also need you to sign the CLA (see the first comment in the PR).

**Pull request closed without merging**